packages/data-context/src/data/ProjectConfigIpc.ts

```diff
@@ -96,7 +107,8 @@ export class ProjectConfigIpc extends EventEmitter {
   }
 
   on(evt: 'childProcess:unhandledError', listener: (err: CypressError) => void): this
-
+  on(evt: 'export:telemetry', listener: (data: string) => void): this
```

Comment:
anything using the value returned?
```suggestion
  on(evt: 'export:telemetry', listener: (data: string) => void): void
```